doc/cloud-init.md

```diff
@@ -33,6 +33,7 @@ Rebooting the instance does not re-trigger the actions.
 To use `cloud-init`, you must base your instance on an image that has `cloud-init` installed:
 
 * All images from the `ubuntu` and `ubuntu-daily` {ref}`image servers <remote-image-servers>` have `cloud-init` support.
+* Images for Ubuntu releases prior to `20.04` from the `ubuntu` and `ubuntu-daily` image servers require special handling to integrate properly with `cloud-init` and have `lxc exec` working with virtual machines using those images. Refer to {ref}`vm-cloud-init-config`.
```

Comment:
```suggestion
 However, images for Ubuntu releases prior to `20.04` require special handling to integrate properly with `cloud-init`, so that `lxc exec` works correctly with virtual machines that use those images. Refer to {ref}`vm-cloud-init-config`.
```

Reply:
(This shouldn't be a separate bullet point, since it's a limitation/clarification of the first one.)